packages/riipen-ui/src/components/Avatar.test.jsx

```diff
@@ -0,0 +1,167 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Avatar from "./Avatar";
+
+describe("<Avatar>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Avatar />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Avatar />);
+
+      expect(wrapper.find("Avatar").props().size).toEqual("96px");
+      expect(wrapper.find("Avatar").props().variant).toEqual("circle");
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const wrapper = mount(<Avatar />);
+
+      expect(wrapper.find("Avatar").props().classes).toEqual([
+        "riipen",
+        "riipen-avatar"
+      ]);
+    });
+  });
+
+  describe("alt prop", () => {
+    it("sets inner class name given custom alt props", () => {
+      const altVariant = "Test String";
+
+      const wrapper = mount(<Avatar alt={altVariant} />);
+
+      expect(
+        wrapper
+          .find(".avatar")
+          .childAt(0)
+          .hasClass("inner")
+      ).toEqual(true);
+      expect(wrapper.find("Avatar").props().alt).toEqual(altVariant);
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const initials = "ZZ";
+      const child = <div>{initials}</div>;
+
+      const wrapper = mount(<Avatar>{child}</Avatar>);
+
+      expect(
+        wrapper
+          .find(".inner")
+          .children()
+          .contains(child)
+      ).toEqual(true);
+      expect(
+        wrapper
+          .find(".inner")
+          .childAt(0)
+          .text()
+      ).toEqual(initials);
```

Comment:
If you simplify the above, you can remove this test. This test is just a subset of the one above it, so we are actually testing the same thing twice.